Sources/Build/BuildOperation.swift

```diff
@@ -136,10 +136,14 @@ public final class BuildOperation: PackageStructureDelegate, SPMBuildCore.BuildS
         let oldBuildPath = buildParameters.dataPath.parentDirectory.appending(
             component: buildParameters.configuration.dirname
         )
-        if localFileSystem.exists(oldBuildPath) {
-            try localFileSystem.removeFileTree(oldBuildPath)
+        do {
+            if localFileSystem.exists(oldBuildPath) {
+                try? localFileSystem.removeFileTree(oldBuildPath)
```

Comment:
with this change any error in attempting to delete the oldBuildPath is ignored. intentional?

Reply:
Yes.  In most cases deletion error will also lead to symlink error (if not, there is actually nothing wrong), so I ignored it here.

Reply:
Since this is already in a `do` `catch` block, we should not need to ignore the error.

```suggestion
                try localFileSystem.removeFileTree(oldBuildPath)
```

Reply:
This may lead to confusion because the warning here indicates an error in symlinking instead of deleting.

If we want to handle it, there should be another nested `do-catch`, which I don’t think necessary🤔

Reply:
we can change the warning to say "unable to delete \(oldBuildPath) or create symbolic link at \(oldBuildPath): \(error)"